k8s/base/merge-orchestrator-server-deployment.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: merge-orchestrator-server
+  labels:
+    name: merge-orchestrator-server
+    version: "1"
+spec:
+  selector:
+    matchLabels:
+      name: merge-orchestrator-server
+  template:
+    metadata:
+      labels:
+        name: merge-orchestrator-server
+    spec:
+      containers:
+      - name: merge-orchestrator-server
+        image: docker.io/gridsuite/merge-orchestrator-server:latest
+        imagePullPolicy: Always
```

Comment:
to be set to IfNotPresent if Jon's PR will be merged before that PR 
@jonenst

Reply:
@FranckLecuyer Jon has merged his PR so that modification is now required

Reply:
Modification done